specification/servicefabric/data-plane/readme.md

```diff
@@ -60,6 +60,15 @@ input-file:
 ```
 
 
+### Tag: 6.1
```

Comment:
Please add this to Python conf (or what Python version you want actually)
```
package-version: 6.1.2.9
```

Reply:
Can we have the python package match the spec version? So in this case 6.1.2.9?

Reply:
It looks like it does match already, I think that's okay

Reply:
@samedder It was longer to explain than to do it, so I pushed a commit in your PR :)

Reply:
I see, is there any issue with using `6.1.2` instead of `6.1.2.9` ?